VoiceMeeterPlugin/Commands/Bases/SingleBaseAdjustment.cs

```diff
@@ -86,12 +82,14 @@ private void GetNewSettings()
                     (Int32)Remote.GetParameter(
                         $"{(this.IsStrip ? "Strip" : "Bus")}[{hiIndex + this.Offset}].{this.Command}");
             }
-
+
+            if (loaded)
             this.AdjustmentValueChanged();
         }
-
+        protected bool loaded;
```

Comment:
It would be good, if you could move this variable to the others, also respect the naming convention please.